examples/script/multi_arch_sym.py

```diff
@@ -35,6 +35,3 @@ def entered_func(state):
     m.verbosity = 2
```

Comment:
while we're here, can we clean up the `start()` typo on the above line (also in the other script)? maybe we can even remove "start() returns when Manticore finishes" and put something equivalent in the API docs.

Reply:
Was adding something similar to the API docs, then took a moment -- isn't this implied?